src/main/kotlin/org/jitsi/jicofo/JicofoServices.kt

```diff
@@ -201,7 +184,7 @@ open class JicofoServices {
             it.stop()
         }
         healthChecker?.stop()
-        channelAllocationExecutor.shutdownNow()
+        TaskPools.ioPool.shutdownNow()
```

Comment:
I feel like it'd be cleaner to have a single shutdown method in the `TaskPools` companion object.